specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -492,6 +500,155 @@
           }
         }
       }
+    },
+    "/weather/airQuality/current/{format}": {
+      "get": {
+        "description": "**Get Current Air Quality**\n\n**Applies to**: S1 and G2 pricing tiers.\n\nThe service provides the detailed information about the concentration of pollutant and overall status for current air quality.",
+        "operationId": "Weather_GetCurrentAirQuality",
+        "x-ms-examples": {
+          "CurrentAirQuality": {
```

Comment:
You may also consider friendly names for those example: like https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Render/preview/2.1/render.json#L61

Reply:
Updated